docs/source/index.rst

```diff
@@ -130,6 +130,7 @@ Documentation
    reference/faq
    reference/logging
    reference/local/index
+   reference/add-gpu
```

Comment:
It would be nice to have another section called `Developer Guides` at the bottom of the doc containing this page.